app/com/gu/contentapi/sanity/package.scala

```diff
@@ -23,10 +23,11 @@ package object sanity extends ScalaFutures with Matchers with IntegrationPatienc
   def incrementIncidentCount = {
     incidentCount += 1
```

Comment:
@cb372 you're right that's how it would work currently, my feeling is that that's 'good enough' currently and we can revisit if we start to see erroneous reports because of this logic.